src/Runner.Listener/MessageRef.cs

```diff
@@ -0,0 +1,11 @@
+using System.Runtime.Serialization;
+
+namespace GitHub.Runner.Listener
+{
+    [DataContract]
+    public sealed class MessageRef
```

Comment:
`RunnerJobRequestRef`?

Reply:
I like that. This is specific to job requests and not for example cancel messages